README.md

```diff
@@ -1,20 +1,24 @@
+---
+title: Running Weave Kubernetes Anywhere
+layout: default
+---
+
```

Comment:
I don't think this is needed at this point in time, but let's discuss when you are online